src/jobs/js_build_job.yml

```diff
@@ -3,7 +3,35 @@ description: >
 
 # Executor
 
-executor: js
+executor: << parameters.e >>
+
+# Parameters
+
+parameters:
+  app_type:
+    default: nextjs
+    enum: ["legacy", "nextjs"]
+    type: enum
+  build_env:
+    default: beta
+    enum: ["beta", "prod"]
+    type: enum
+  e:
```

Comment:
Can we be more verbose here and name it executor? To make configs easier to read for others.